src/app/meeting/write/page.tsx

```
@@ -0,0 +1,199 @@
+            <input
+              value={title}
+              onChange={(e) => setTitle(e.target.value)}
+              maxLength={50}
+              placeholder="모임명을 입력하세요 (예: 2026 봄학기 신입 환영회)"
+              className="w-full rounded-xl bg-gray-50 px-5 py-4 text-base text-gray-900 placeholder-gray-400 focus:outline-none"
+            />
+
+            <div className="mt-6 grid grid-cols-1 gap-6 md:grid-cols-2">
+              {/* 카테고리 */}
+              <div>
+                <label className="mb-2 block text-sm font-semibold text-gray-900">카테고리</label>
```

Biome (2.5.0)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)